python/mxnet/ndarray.py

```diff
@@ -953,7 +953,7 @@ def ones(shape, ctx=None, dtype=mx_real_t):
 
     Parameters
     ----------
-    shape : int or tuple of int
+    shape : int or sequence of ints
```

Comment:
Actually I took the numpy.ones as reference. 
https://docs.scipy.org/doc/numpy/reference/generated/numpy.ones.html
The `sequence` should represents both `list`, `tuple`.